test/clj/cider/nrepl/middleware/apropos_test.clj

```diff
@@ -9,6 +9,14 @@
 
 (use-fixtures :each session/session-fixture)
 
+(deftest msg->var-query-map-test
+  (testing "Constructs the ns-query map correctly"
+    (let [msg {:exclude-regexps ["^cider.nrepl" "^refactor-nrepl" "^nrepl"]
+               :query "spelling"}
+          query-map (#'apropos/msg->var-query-map msg)]
+      (is (contains? (:var-query query-map) :ns-query))
```

Comment:
Won't it be better to just compare the result with the expected map?

Reply:
i thought about that but i think the returned map will have regex's rather than strings for the exclude-regexps and I wasn't sure of the semantics of comparing regexes

Reply:
and it appears they compare by reference. `(= #"bob" #"bob")` is false. so it would require some wrangling that i wasn't into at the time :) I can take another crack at it later if you like.

Reply:
Got it. It's fine like this.